src/app.js

```diff
@@ -84,15 +94,15 @@ function startHttpsGateway() {
       addonManager.loadAddons();
     });
     rulesEngineConfigure(httpsServer);
-    console.log('Listening on port', httpsServer.address().port);
+    console.log('HTTPS server listening on port ', httpsServer.address().port);
     commandParserConfigure(httpsServer);
   });
 
   // Redirect HTTP to HTTPS
   httpServer.on('request', createRedirectApp(httpsServer.address().port));
   const httpPort = config.get('ports.http');
   httpServer.listen(httpPort, function() {
-    console.log('Redirector listening on port', httpServer.address().port);
+    console.log('*** Redirector listening on port ', httpServer.address().port);
```

Comment:
Why the changes to this log line (and the next, on line 123)?

Reply:
A habit from another age.... Done.